packages/messages/src/browser/notifications-message-client.ts

```diff
@@ -35,14 +37,24 @@ export class NotificationsMessageClient extends MessageClient {
             label: action,
             fn: element => onCloseFn(action)
         });
+
+        // If some actions are defined, we don't set the timeout
```

Comment:
convert to JSDoc, please.

Reply:
Using macro does not need anymore the comment